.github/workflows/eden.yml

```diff
@@ -59,23 +59,27 @@ jobs:
         arch: [amd64]
         hv: [kvm]
     if: github.event.review.state == 'approved'
-    uses: lf-edge/eden/.github/workflows/test.yml@0.9.11
+    uses: lf-edge/eden/.github/workflows/test.yml@1.0.2
+    secrets: inherit
     with:
       eve_image: "evebuild/pr:${{ github.event.pull_request.number  }}"
-      eve_artifact_name: eve-${{ matrix.hv }}-${{ matrix.arch }}
-      artifact_run_id: ${{ needs.get-run-id.outputs.result }}
-      eden_version: "0.9.11"
+      eve_log_level: "debug"
+      eve_artifact_name: eve-${{ matrix.hv }}-${{ matrix.arch }}-${{ matrix.platform }}
```

Comment:
Do we need this change of the matrix here? Does it work well with the rest of the actions in 12.0?